compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/HttpFiltersDesugar.java

```diff
@@ -88,47 +95,51 @@
         this.names = Names.getInstance(context);
     }
 
-    boolean isHttpPackage(List<BType> expressionTypes) {
+    void desugarFunction(BLangFunction functionNode, SymbolEnv env, List<BType> expressionTypes) {
+        if (isValidPackage(expressionTypes, HTTP_PACKAGE_NAME)) {
+            addHttpAuthDesugarFunctionInvocation(functionNode, env);
+        } else if (isValidPackage(expressionTypes, WEBSOCKET_PACKAGE_NAME)) {
+            addWebSocketAuthDesugarFunctionInvocation(functionNode, env);
+        }
+    }
+
+    boolean isValidPackage(List<BType> expressionTypes, String packageName) {
```

Comment:
I this this should be something like `isDefinedInStdLibPackage`

Reply:
Yes. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/31045/commits/f29c2b1d5fdd13cce00ba38fb2be681a07a5ea65